docs/docs/guides/web3_plugin_guide/index.md

```diff
@@ -1,11 +1,48 @@
 ---
-sidebar_position: 6
-sidebar_label: 'Plugins'
+sidebar_position: 1
+sidebar_label: 'Introduction'
 ---
 
-# web3.js Plugins Guide
+# Introduction
 
-In addition to the web3.js standard libraries, plugins add specific functionality to the end user. This extra functionality could be wrappers around specific contracts, additional RPC method wrappers, or could even extend the logic of web3.js methods.
+Welcome to the web3.js Plugins Guide, an exciting new feature introduced in web3.js v4! In addition to the core web3.js libraries, plugins bring specialized functionalities tailored for end-users (functionalities that you, as a developer, can create). These enhancements may involve creating wrappers for specific contracts, adding extra features to RPC methods, or extending the capabilities of web3.js methods. Dive in and explore this innovative addition to web3.js v4!
 
-- [web3.js Plugin Developer Guide](/guides/web3_plugin_guide/plugin_authors)
-- [Plugin User Guide](/guides/web3_plugin_guide/plugin_users)
\ No newline at end of file
+- [Plugin Developer Guide (For Creators)](/guides/web3_plugin_guide/plugin_authors)
+- [Plugin User Guide (Usage)](/guides/web3_plugin_guide/plugin_users)
+
+## Plugin Showcase
+
+### Chainlink Plugin
+- [`npm i @chainsafe/web3-plugin-chainlink`](https://www.npmjs.com/package/@chainsafe/web3-plugin-chainlink)
+- **Description**: A Web3.js 4.x Plugin for Interacting With Chainlink Smart Contracts
+- **Author**: ChainSafe Systems
+
+### Tokens Plugin
+- [`npm i @chainsafe/web3-plugin-tokens`](https://www.npmjs.com/package/@chainsafe/web3-plugin-tokens)
+- **Description**: Plugin to extend web3.js with additional methods to interact with common token interfaces (ERC20, ERC721, ERC1155...)
+- **Author**: Peter Grassberger & ChainSafe
+
+### Craftsman Plugin
+- [`npm i web3-plugin-craftsman`](https://www.npmjs.com/package/web3-plugin-craftsman)
+- **Description**: web3.js plugin allowing instantiation of contract objects directly from Solidity source code
+- **Author**: Muhammad-Altabba
+
+### Optimism Plugin
+- [`npm i @eth-optimism/web3.js-plugin`](https://www.npmjs.com/package/@eth-optimism/web3.js-plugin)
+- **Description**: Web3.js plugin for OP-Chain gas estimation
+- **Author**: Unknown
+
+### Near Protocol Plugin
+- [`npm i @conx3/web3-plugin-near`](https://npmjs.com/package/@conx3/web3-plugin-near)
+- **Description**: web3.js plugin for Near Protocol
+- **Author**: Muhammad Altabba
+
+### Aurora Engine Plugin
+- [`npm i @conx3/web3-plugin-aurora`](https://www.npmjs.com/package/@conx3/web3-plugin-aurora)
+- **Description**: web3.js plugin for Aurora Engine, an EVM running atop NEAR protocol
+- **Author**: Muhammad Altabba
+
+### Superfluid Plugin
```

Comment:
This looks good, but when new plugin will be added we will need to maintain this manually along with PR by contributor in web3.js website. so could you find way of automating this list using https://github.com/web3/web3js-landing/blob/main/src/pluginList.ts